public/index.html

```diff
@@ -24,8 +36,8 @@
       You can add webfonts, meta tags, or analytics to this file.
       The build step will place the bundled scripts into the <body> tag.
 
-      To begin the development, run `npm start`.
-      To create a production bundle, use `npm run build`.
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
```

Comment:
None of these changes have anything to do with TypeScript. The same goes for the images that have been replaced/removed.

Reply:
It has to do with the generation of the TypeScript stuff (I finally did it, but it was a pain in the ass). 
I'll try to replace the images

Reply:
done...